### DISCUSSION TITLE
headphones get confused cross platform

### DISCUSSION THREAD

**Discussion by kaounasnik** [blueman-project/blueman#2546] [Q&A]
Not sure if this is an app bug or entirely related to the nature of the operating systems, my headphones do weird things (like not showing up in search, not showing up as an  audio device, not being able to pair or connect) when i switch to windows 11 or viseversa. 
My troubleshooting is a combination of on/off the headphones and the bluetooth service on  my machine.

**Comment by infirit** [comment]
Generally you need make Bluetooth devices discoverable and pairable.  How this is done depends on the device but usually you hold a button for a couple seconds.

An issue you generally can run into when dualbooting is that the pairing authentication doesn't match. Which means you'll have to redo the pairing when switching operating system.
